src/Symfony/Component/Form/Extension/Core/Type/FormFlowNavigatorType.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Flow\FormFlowCursor;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * A navigator type that defines default actions to interact with a form flow.
+ *
+ * @author Yonel Ceruto <open@yceruto.dev>
+ */
+class FormFlowNavigatorType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder->add('back', FormFlowActionType::class, [
+            'action' => 'back',
+            'include_if' => fn (FormFlowCursor $cursor): bool => $cursor->canMoveBack(),
+        ]);
+
+        $builder->add('next', FormFlowActionType::class, [
+            'action' => 'next',
+            'include_if' => fn (FormFlowCursor $cursor): bool => $cursor->canMoveNext(),
+        ]);
+
+        $builder->add('finish', FormFlowActionType::class, [
```

Comment:
"last" ?

according to previous/next, last would be in the same naming logic, finish would more likely to be an action

Reply:
Same as https://github.com/symfony/symfony/pull/60212#discussion_r2045260968